Documentation/cold-staking-setup.md

```diff
@@ -0,0 +1,65 @@
+## COLD STAKING SETUP INSTRUCTIONS
+
+**What you will need** 
+
+ - **Hot wallet** - this will be used for staking online, you can skip this if you are using _www.trustaking.com_
+ - **Cold wallet** - this will hold your funds offline
+   
+_Please note. It's important that both wallets are running on separate full nodes._
```

Comment:
It's important that both wallets are used with separate full nodes instances, and are not present in the same data folder.

Reply:
Updated